app/main.py

```diff
@@ -1,3 +1,16 @@
-def cache(func):
-    # Write your code here
-    pass
+def cache(func):
+    data = dict()
+
+    def wrapper(*args):
+        nonlocal data

```

Comment:
this statement is redundant here, because you are not rewriting data variable, you just modify dict that is under this variable